tests/foreman/api/test_reporttemplates.py

```diff
@@ -591,3 +591,65 @@ def test_positive_schedule_entitlements_report(setup_content, target_sat):
         )
         assert vm.hostname in data_csv
         assert DEFAULT_SUBSCRIPTION_NAME in data_csv
+
+
+@pytest.mark.tier3
+def test_positive_generate_job_report(setup_content, target_sat):
+    """Generate a report using the Job - Invocation Report template.
+
+    :id: 946c39db-3061-43d7-b922-1be61f0c7d93
+
+    :bz: 1761012
+
+    :steps:
+
+        1. Register a host and properly setup REX for it.
+        2. Run a simple job with predictable output
+        3. Using the Job ID, generate a report using the Job - Invocation
+           report template.
+
+    :expectedresults: Report returns correct information (Hostname is set correctly,
+        the output is what would be expected.)
+
+    :CaseImportance: High
+    """
+    with Broker(nick='rhel7', host_class=ContentHost) as vm:
```

Comment:
You can directly use `rhel7_contenthost` fixture here https://github.com/SatelliteQE/robottelo/blob/21254ccbd16a55d12c84302a2a803b471904bf0c/pytest_fixtures/core/contenthosts.py#L50